File: Data Structures & Algorithms/longest-palindromic-substring/submission-1.py
class Solution:
    def longestPalindrome(self, s: str) -> str:


        longest = ""
        for j in range (0, len(s)):
            curr = ""
            for i in range (j, len(s)):
                curr += s[i]
                if curr == curr[::-1]:
                    if len(curr) == max(len(longest), len(curr)):
                        longest = curr
        return longest
        

File: Data Structures & Algorithms/maximum-subarray/submission-1.java
class Solution {
    public int maxSubArray(int[] nums) {

        int curr = nums[0];
        int maxSum = nums[0];
        for(int i =1; i<nums.length; i++){
            if(curr<0){
                curr = 0;
            }
            curr += nums[i];
            maxSum = Math.max(curr, maxSum);
        }

        return maxSum;
    }
}
